pkg/plugins/golang/multi-module/v1alpha1/gomod.go

```diff
@@ -0,0 +1,92 @@
+package v1alpha1
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/afero"
+	"sigs.k8s.io/kubebuilder/v3/pkg/config"
+	"sigs.k8s.io/kubebuilder/v3/pkg/machinery"
+	"sigs.k8s.io/kubebuilder/v3/pkg/plugin/util"
+	v3scaffolds "sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/v3/scaffolds"
+)
+
+const (
+	ControllerRuntime        = "sigs.k8s.io/controller-runtime"
+	ControllerRuntimeVersion = v3scaffolds.ControllerRuntimeVersion
+	GoVersion                = "1.18"
```

Comment:
Why this plugin needs the C+R version and the Go one?
Can not we try to do something here that will make it eaiser if we keep it maintained? 

Why not use the same versions defined in the go plugin, see that for example the kustomize version is defined in the kustomize plugin ( it is part of its own domain of responsibility ) but the go base plugin needs to scaffold the makefile accordingly and use it: 

https://github.com/kubernetes-sigs/kubebuilder/blob/1502ede9731d67c591c8a9d3620d6a525ea49953/pkg/plugins/golang/v3/scaffolds/init.go#L104-L110

Can we not to do something similar here?